cpp/src/slice2java/Gen.cpp

```diff
@@ -4071,6 +4074,9 @@ Slice::Gen::TypesVisitor::visitDictionary(const DictionaryPtr& p)
     out << eb;
     out << eb;
 
+    out << sp;
+    out << nl << "private " << name << "Helper() {}";
+
```

Comment:
This is for the dictionary helper class.

Same thing, make it impossible to construct an instance of this fully static helper class.